### PR TITLE
Fix go version in dockerfile

### PR DESCRIPTION
**Pull request by ischasny**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18-alpine AS builder
+FROM golang:1.19-bullseye AS builder
```
Comment (gammazero) [inline Dockerfile:1]: Should build with 1.20

Comment (ischasny) [reply Dockerfile]: Fixed

File: .github/workflows/container-build.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Container
+
+on: [ push, pull_request ]
+
+jobs:
+  publisher:
+    name: Build
+    runs-on: ubuntu-latest
```
Comment (gammazero) [inline .github/workflows/container-build.yml:8]: I wonder if ubuntu is the best choice here.  Seems like a lot of baggage.

**Review by gammazero — APPROVED** [review]

File: .github/workflows/container-build.yml (hunk 2 above)
Comment (ischasny) [reply .github/workflows/container-build.yml:8]: I just restore the file that I deleted before

**Pull request merged**